src/content/docs/de/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,679 @@
+
+Siehe [das Änderungsprotokoll](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) für die vollständigen Versionshinweise.
+
+### Entfernt: Unterstützung für Node 16
+
+Node 16 wird voraussichtlich im September 2023 sein Lebensende erreichen.
+
+Astro v3.0 verzichtet komplett auf die Unterstützung von Node 16, damit alle Astro-Benutzer die Vorteile der moderneren Features von Node nutzen können.
+
+#### Was soll ich tun?
+
+Vergewissere dich, dass sowohl deine Entwicklungs- als auch dein Veröffentlichungs-Umgebung **Node `18.14.1` oder höher** verwenden.
+
+1. Überprüfe deine lokale Version von Node mit:
+    ```sh
+    node -v
+    ```
+
+
+
+2. Überprüfe die Dokumentation deiner [Veröffentlichungs-Umgebung](/de/guides/deploy/), um sicherzustellen, dass sie Node 18 unterstützt.
+
+Du kannst Node `18.14.1` für dein Astro-Projekt entweder in einer Dashboard-Konfigurationseinstellung oder in einer `.nvmrc`-Datei angeben.
+
+```bash title=".nvmrc"
+18.14.1
+```
+### Entfernt: Unterstützung für TypeScript 4
+
+Die `tsconfig.json`-Voreinstellungen in Astro v2.x Unterstützen TypeScript 4.x und 5.x. Astro geht jetzt davon aus, dass du TypeScript 5.0 (März 2023) verwendest oder das dein Editor diese Version einschließt (wie z.B. VS Code 1.77).
+
+#### Was soll ich tun?
+
+Falls du TypeScript lokal installiert hast, aktualisiere es mindestens auf v5.0.
+
+```bash
+npm install typescript@latest --save-dev
+```
+
+### Entfernt: `@astrojs/image`
+
+In Astro v2.x stellte Astro eine offizielle Bildintegration zur Verfügung, die Astro `<Image />`- und `<Picture />`-Komponenten einschloss.
+
+Mit Astro v3.0 wurde diese Integration komplett entfernt. Die neue Lösung für den umgang mit Bildern ist eine integrierte Bildservices-API: `astro:assets`.
+
+#### Was soll ich tun?
+
+Entferne die `@astrojs/image`-Integration aus deinem Projekt. Du musst nicht nur die Integration deinstallieren, sondern auch alle Import-Anweisungen sowie die vorhandenen `<Image />`- und `<Picture />`-Komponenten aktualisieren oder entfernen. Möglicherweise musst du auch einen bevorzugten Standardbildverarbeitungsdienst konfigurieren.
+
+Du findest eine [Schritt für Schritt Anleitung zum Entfernen der alten Bildintegration](/de/guides/images/#remove-astrojsimage) in unserem Bilder-Guide.
+
+Die Migration zu `astro:assets` bringt auch einige neue Bildoptionen und Features mit sich, die du vielleicht nutzen möchtest. Bitte sieh dir den vollständigen [Upgrade-Ratgeber für Bilder in Version 3.0](/de/guides/images/#upgrade-to-v30-from-v2x) für alle Details an!
+
+```js del={3,7}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import image from '@astrojs/image';
+
+export default defineConfig({
+  integrations: [
+    image(),
+  ]
+})
+```
+
+### Entfernt: `<Markdown />`-Komponente
+
+In Astro v1.x wurde der <Markdown />-Komponent veraltet und in ein externes Paket verschoben.
```

Comment:
```suggestion
In Astro v1.x wurde der `<Markdown />`-Komponent veraltet und in ein externes Paket verschoben.
```
I think this will fix the problem with deploying!